pipelines/cluster-role-binding.yaml

```diff
@@ -0,0 +1,12 @@
+piVersion: rbac.authorization.k8s.io/v1
```

Comment:
Regarding the directory structure, since this is our first pass a the "development" pipeline, let's nest the files under something like pipelines/development.

Reply:
I put the files that were related to the dev pipeline itself under a dev-pipeline dir.  The couple of files at the top level I thought were generic and were applicable to multiple pipelines.  If you think they are not, let me know and I can move it.